src/components/FoodForm.js

```diff
@@ -0,0 +1,103 @@
+import {nanoid} from 'nanoid';
+import {useState} from 'react';
+
+import {StyledDoneLabel} from './styled/StyledDoneLabel';
+import {StyledForm} from './styled/StyledForm';
+import {StyledInput} from './styled/StyledInput';
+import {StlyedList} from './styled/StyledList';
+import {StyledListItems} from './styled/StyledListItems';
+import {StyledWidthBox} from './styled/StyledWidthBox';
```

Comment:
You could use a default exprot here, it would save you some {}.
just a suggestion, this works as well 👍

Reply:
> You could use a default exprot here, it would save you some {}. just a suggestion, this works as well 👍

I think the code looks more cleaner if all imports are named or default ones and I decided here to use named Imports to get used to it. but I appreciate your suggestion, thank you.